CHANGELOG.md

```diff
@@ -13,13 +13,14 @@ changes.
 - Direct chain integration which allows to connect to a real cardano-node /
   devnet; no on-chain validators though.
 - Support alonzo transactions inside the Hydra Head. For now using a `freeCostModel`.
-- Command line options `--node-socket`, `--network-magic` and
+- Command line options `--cardano-node-socket`, `--network-magic` and
   `--cardano-{signing,verification}-key` to `hydra-node` and `hydra-tui` to
```

Comment:
AFAICS the `hydra-node` uses `--node-socket` and the `hydra-tui` uses `--cardano-node-socket` currently. 

Should: Make it more consistent by naming this `--node-socket` on both or `--cardano-node-socket` on both.